### PR TITLE
Update/logstash

### PR DESCRIPTION
**Pull request by dontfight**
1. accept LOG_LEVEL env.
2. clean up for logstash format for better Kibana reporting.

### CODE REVIEW

File: healthchain-logger.js [hunk 1]
```diff
@@ -21,57 +23,56 @@ let Logger = {
...
-    this.logger = createLogger({
-      transports: [this.config.transport]
-    });
-    this.requestLogger = morgan(this.config.morganFormat, { stream: this.stream });
   },
   requestId: (req, res, next) => {
-    const requestId = req.headers['x-request-id'] || uuid.v4();
-    Logger.REQUEST_ID = requestId;
-    res.setHeader('x-request-id', requestId);
+    xrequestId = req.headers['x-request-id'] || requestId;
```
Comment (bsteinfeld) [inline healthchain-logger.js]: should this be `const`

File: healthchain-logger.js [hunk 2]
```diff
@@ -21,57 +23,56 @@ let Logger = {
...
-    this.logger = createLogger({
-      transports: [this.config.transport]
-    });
-    this.requestLogger = morgan(this.config.morganFormat, { stream: this.stream });
   },
   requestId: (req, res, next) => {
-    const requestId = req.headers['x-request-id'] || uuid.v4();
-    Logger.REQUEST_ID = requestId;
-    res.setHeader('x-request-id', requestId);
+    const xRequestId = req.headers['x-request-id'] || requestId;
```
Comment (egorbenko) [inline healthchain-logger.js]: does this work? I don't think it will create new uuid on each request this way

Comment (dontfight) [reply healthchain-logger.js]: fixed

File: healthchain-logger.js (hunk 1 above)
Comment (dontfight) [reply healthchain-logger.js]: fixed

**Pull request merged**